egui/src/widgets/plot/mod.rs

```diff
@@ -446,35 +459,43 @@ impl PlotUi {
         Hsva::new(h, 0.85, 0.5, 1.0).into() // TODO: OkLab or some other perspective color space
     }
 
-    /// The pointer position in plot coordinates, if the pointer is inside the plot area.
+    pub fn ctx(&self) -> &CtxRef {
+        self.ctx
+    }
+
+    /// The plot bounds as they were in the last frame.
```

Comment:
What is its value the first frame? Would it make more sense to return `None` the first frame?

Reply:
I guess the same comment applies to `screen_from_plot` and `plot_from_screen` - what is apropiate to return the first frame?

Reply:
We made sure that the bounds never become invalid. So if no items have been added to the plot and the bounds were not specified in the builder, they will at least be in the valid range `[-1.0, 1.0]`. The first transform may not be very accurate, but valid and I prefer this to making the user deal with an `Option` just because of one frame. But maybe you know an instance where this could go wrong?

Reply:
Good point, I added some documentation.